node/node_handler.go

```diff
@@ -360,7 +360,7 @@ func (node *Node) PostConsensusProcessing(newBlock *types.Block, commitSigAndBit
 		// 15% of the validator also need to do broadcasting
 		rand.Seed(time.Now().UTC().UnixNano())
 		rnd := rand.Intn(100)
-		if rnd < 1 {
+		if rnd < 0 {
```

Comment:
so this is where we turn off for non-leaders

Reply:
yes